test/relayclient/GSNConfigurator.test.ts

```diff
@@ -0,0 +1,105 @@
+// test possible client errors
+
+import { GsnTestEnvironment, TestEnvironment } from '../../src/relayclient/GsnTestEnvironment'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import { resolveConfigurationGSN } from '../../src/relayclient/GSNConfigurator'
+import { constants } from '../../src/common/Constants'
+import { DeploymentResult } from '../../src/cli/CommandsLogic'
+import { PrefixedHexString } from 'ethereumjs-tx'
+import ContractInteractor from '../../src/relayclient/ContractInteractor'
+
+const { assert, expect } = chai.use(chaiAsPromised)
+
+contract('client-configuration', () => {
+  let env: TestEnvironment
+  let deploymentResult: DeploymentResult
+  let paymasterAddress: PrefixedHexString
+  before(async () => {
+    env = await GsnTestEnvironment.startGsn('http://localhost:8544')
+    deploymentResult = env.deploymentResult
+    // deploymentResult = loadDeployment('./build/gsn')
+    paymasterAddress = deploymentResult.naivePaymasterAddress
+  })
+  describe('#resolveConfigurationGSN', () => {
+    describe('failures', () => {
+      it('should fail with no params', async () => {
+        // @ts-ignore
+        await expect(resolveConfigurationGSN()).to.eventually.rejectedWith(/Cannot read property/)
+      })
+
+      it('first arg not provider', async () => {
```

Comment:
Please take this line:
`should throw if the`
and use Command-C + Command-V to make these dest descriptions more readable in isolation.